server/historian/packages/historian-base/src/app.ts

```diff
@@ -49,6 +51,7 @@ export function create(
                 eventName: "http_requests",
              };
              winston.info("request log generated", { messageMetaData });
+             Lumberjack.log("request log generated", LogLevel.Info, messageMetaData);
```

Comment:
Can you add another property called `telemetryGroupName` in the `properties` parameter to `Lumberjack`? Since `Lumberjack` internally uses a `Lumber` object for logging, and since `Lumber` already has a specific property called `eventName`, we have been using this `telemetryGroupName` property (available as `CommonProperties.telemetryGroupName`)  instead.
```typescript
Lumberjack.log(
        message,
        LogLevel.Info,
        {
            ...messageMetaData,
            [CommonProperties.telemetryGroupName]: messageMetaData.eventName,
        });
```